plugins/Toolbox/src/Toolbox.py

```diff
@@ -226,38 +218,27 @@ def _onAppInitialized(self) -> None:
         # On boot we check which packages have updates.
         if CuraApplication.getInstance().getPreferences().getValue("info/automatic_update_check") and len(installed_package_ids_with_versions) > 0:
             # Request the latest and greatest!
-            self._fetchPackageUpdates()
+            self._makeRequestByType("updates")
         self._fetchUserSubscribedPackages()
 
-    def _prepareNetworkManager(self):
```

Comment:
If we are deleting this from here, shouldn't we also delete the called functions `_onRequestFinished` and `_onNetworkAccessibleChanged`, since they are not used and just can confuse people around?
They are already in the PackageManager in Uranium.

Reply:
Yeah, it seems these are not used, I agree